index.html

```diff
@@ -0,0 +1,55 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="container">
+        <div class="wrapper">
```

Comment:
- [ ] Good job👌, but kindly try to use the correct tags for semantics and user experience, so please wrap the logo and the nav links in the future and the hamburger inside the ```<header>``` tag, and the other things which you put it inside a class box2 as p and socials  wrap them inside the ```<section>``` tag 💯